### PR TITLE
Add client info traffic stat fields.

### PR DESCRIPTION
**Pull request by CharlesChen888**
Corresponding doc PR to valkey-io/valkey#327.

* `tot-net-in`: Total network input bytes read from this client. Added in Valkey 8.0
* `tot-net-out`: Total network output bytes sent to this client. Added in Valkey 8.0
* `tot-cmds`: Total count of commands this client executed. Added in Valkey 8.0

### CODE REVIEW

File: commands/client-list.md [hunk 1]
```diff
@@ -34,6 +34,11 @@ Here is the meaning of the fields:
 * `user`: the authenticated username of the client
 * `redir`: client id of current client tracking redirection
 * `resp`: client RESP protocol version.
+* `lib-name`: The client library name as set by `CLIENT SETINFO`
+* `lib-version`: The client library version as set by `CLIENT SETINFO`
+* `tot-net-in`: Total network input bytes read from this client. Added in Valkey 8.0
```
Comment (stockholmux) [inline commands/client-list.md:39]: So `'lib-name` and `lib-version` work today, but the `tot-*` won't be added until your PR is merged and 8.0 is released? I feel like we should seperate this into two PRs. One we can merge today and one that depends on valkey-io/valkey#327 and 8.0 release.

**Review by stockholmux — CHANGES_REQUESTED** [review]
I think we should split this into two PRs.

File: commands/client-list.md (hunk 1 above)
Comment (madolson) [reply commands/client-list.md:39]: I agree.

**Comment by madolson** [comment]
Change LGTM, but agree with stockholmux.

File: commands/client-list.md (hunk 1 above)
Comment (zuiderkwast) [reply commands/client-list.md:39]: There used to be a label for this. `waiting-for-upstream`

**Comment by CharlesChen888** [comment]
`lib-name` and `lib-version` are now separated to #57.

**Review by madolson — APPROVED** [review]

**Pull request merged**